samples/server/petstore/springboot-reactive/src/main/java/org/openapitools/api/FakeApiDelegate.java

```diff
@@ -88,7 +88,7 @@ default Mono<ResponseEntity<OuterComposite>> fakeOuterCompositeSerialize(Mono<Ou
         for (MediaType mediaType : exchange.getRequest().getHeaders().getAccept()) {
             if (mediaType.isCompatibleWith(MediaType.valueOf("*/*"))) {
                 String exampleString = "{ \"my_string\" : \"my_string\", \"my_number\" : 0.8008281904610115, \"my_boolean\" : true }";
-                result = ApiUtil.getExampleResponse(exchange, mediaType, exampleString);
+                result = ApiUtil.getExampleResponse(exchange, MediaType.valueOf("*/*"), exampleString);
```

Comment:
will this be converted to `application/json` under the hood?

Reply:
Good catch 👍
I made some changes to also cover the case pointed out by you and more cases where media type was not set as expected.